beta-src/src/state/game/game-api-slice.ts

```diff
@@ -78,6 +78,17 @@ export const fetchGameMessages = createAsyncThunk(
   },
 );
 
+export const toggleVoteStatus = createAsyncThunk(
+  ApiRoute.GAME_TOGGLEVOTE,
+  async (queryParams: { countryID: string; gameID: string; vote: string }) => {
+    const { data } = await getGameApiRequest(
+      ApiRoute.GAME_TOGGLEVOTE,
+      queryParams,
+    );
+    return data as any;
```

Comment:
Can you setup the Type this will be returned as instead of it being any

Reply:
Nice catch